server/properties/src/config/logback.xml

```diff
@@ -26,63 +26,63 @@
 >
 
   <appender name="FileAppender" class="ch.qos.logback.core.rolling.RollingFileAppender">
-    <file>${gocd.server.log.dir:-logs}/go-server.log</file>
+    <file>${gocd.server.log.dir:-${GO_SERVER_LOG_DIR:-logs}}/go-server.log</file>
```

Comment:
Do we need to check for logs? That seems to be taken care of in `server.sh`

Reply:
Yes, you always seem to keep forgetting windows installers and batch files.

Reply:
Oh, sorry. I didn't forget windows. I had had a conversation with @jyotisingh about making GO_SERVER_SYSTEM_PROPERTIES and GO_AGENT_SYSTEM_PROPERTIES on windows consistent with linux that would eliminate the need for this. That's another conversation. And another PR, if we ever get there.